core/controllers/topic_editor_test.py

```diff
@@ -77,7 +77,13 @@ def setUp(self):
             'old_subtopic_id': None,
             'new_subtopic_id': 1,
             'skill_id': self.skill_id
-        })]
+        }), topic_domain.TopicChange({
+                'cmd': topic_domain.CMD_UPDATE_TOPIC_PROPERTY,
```

Comment:
I think these should all be deindented by 4 (up to the `})]` at the end)

Reply:
Done